<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:image/8.png</normaloff>:image/8.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDraw">
    <property name="title">
     <string>Draw</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPick">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionPick"/>
    <addaction name="actionPalm"/>
   </widget>
   <widget class="QMenu" name="menuFix">
    <property name="title">
     <string>Fit</string>
    </property>
    <addaction name="actionFixcanvas"/>
    <addaction name="actionExpand"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionAdd"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionPrint"/>
    <addaction name="actionPrintPreview"/>
   </widget>
   <widget class="QMenu" name="menuCombination">
    <property name="title">
     <string>Combination</string>
    </property>
    <addaction name="actionCombination"/>
    <addaction name="actionDivide"/>
    <addaction name="actionDivideToEnd"/>
   </widget>
   <widget class="QMenu" name="menuZoom">
    <property name="title">
     <string>Zoom</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionZoomOne"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionMoveToTop"/>
    <addaction name="actionMoveToBottom"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionClearSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuDraw"/>
   <addaction name="menuPick"/>
   <addaction name="menuFix"/>
   <addaction name="menuCombination"/>
   <addaction name="menuZoom"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar_0">
   <property name="windowTitle">
    <string>File</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionAdd"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="actionPrintPreview"/>
   <addaction name="actionPrint"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>Edit</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPick"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionMoveToTop"/>
   <addaction name="actionMoveToBottom"/>
   <addaction name="actionFixcanvas"/>
   <addaction name="actionExpand"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionZoomOne"/>
   <addaction name="actionCombination"/>
   <addaction name="actionDivide"/>
   <addaction name="actionDivideToEnd"/>
   <addaction name="actionPalm"/>
  </widget>
  <widget class="QToolBar" name="toolBar_1">
   <property name="windowTitle">
    <string>InnerShapes</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionPick">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:image/pointer.png</normaloff>:image/pointer.png</iconset>
   </property>
   <property name="text">
    <string>Pick</string>
   </property>
  </action>
  <action name="actionFixcanvas">
   <property name="icon">
    <iconset>
     <normaloff>:image/pic1.png</normaloff>:image/pic1.png</iconset>
   </property>
   <property name="text">
    <string>Fitcanvas</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>:image/open.png</normaloff>:image/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>:image/save.png</normaloff>:image/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset>
     <normaloff>:image/blue-plus-sign-md.png</normaloff>:image/blue-plus-sign-md.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionCombination">
   <property name="icon">
    <iconset>
     <normaloff>:image/group.gif</normaloff>:image/group.gif</iconset>
   </property>
   <property name="text">
    <string>Combination</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset>
     <normaloff>:image/zoomin.png</normaloff>:image/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>ZoomIn</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset>
     <normaloff>:image/zoomout.png</normaloff>:image/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>ZoomOut</string>
   </property>
  </action>
  <action name="actionPalm">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:image/hand-ol-512.png</normaloff>:image/hand-ol-512.png</iconset>
   </property>
   <property name="text">
    <string>Palm</string>
   </property>
  </action>
  <action name="actionZoomOne">
   <property name="icon">
    <iconset>
     <normaloff>:image/zoomone.png</normaloff>:image/zoomone.png</iconset>
   </property>
   <property name="text">
    <string>ZoomOne</string>
   </property>
  </action>
  <action name="actionMoveToTop">
   <property name="icon">
    <iconset>
     <normaloff>:image/arrowup.PNG</normaloff>:image/arrowup.PNG</iconset>
   </property>
   <property name="text">
    <string>MoveToTop</string>
   </property>
  </action>
  <action name="actionMoveToBottom">
   <property name="icon">
    <iconset>
     <normaloff>:image/arrow.png</normaloff>:image/arrow.png</iconset>
   </property>
   <property name="text">
    <string>MoveToBottom</string>
   </property>
  </action>
  <action name="actionDivide">
   <property name="icon">
    <iconset>
     <normaloff>:image/ungroup.gif</normaloff>:image/ungroup.gif</iconset>
   </property>
   <property name="text">
    <string>Divide</string>
   </property>
  </action>
  <action name="actionDivideToEnd">
   <property name="icon">
    <iconset>
     <normaloff>:image/sc_ungroup (2).png</normaloff>:image/sc_ungroup (2).png</iconset>
   </property>
   <property name="text">
    <string>DivideToEnd</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>:image/new.png</normaloff>:image/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normaloff>:image/copy.png</normaloff>:image/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset>
     <normaloff>:image/paste.png</normaloff>:image/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionExpand">
   <property name="icon">
    <iconset>
     <normaloff>:image/pic2.png</normaloff>:image/pic2.png</iconset>
   </property>
   <property name="text">
    <string>Expand</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <normaloff>:image/cut.png</normaloff>:image/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>:image/undo.png</normaloff>:image/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normaloff>:image/redo.png</normaloff>:image/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset>
     <normaloff>:image/save_all_download-512.png</normaloff>:image/save_all_download-512.png</iconset>
   </property>
   <property name="text">
    <string>SaveAs</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset>
     <normaloff>:image/print.png</normaloff>:image/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionPrintPreview">
   <property name="icon">
    <iconset>
     <normaloff>:image/print-preview-icon-54533.png</normaloff>:image/print-preview-icon-54533.png</iconset>
   </property>
   <property name="text">
    <string>PrintPreview</string>
   </property>
  </action>
  <action name="actionClearSettings">
   <property name="icon">
    <iconset>
     <normaloff>:image/clear.png</normaloff>:image/clear.png</iconset>
   </property>
   <property name="text">
    <string>ClearSettings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
